noxfile.py

```diff
@@ -10,7 +10,7 @@
 
 @nox_poetry.session(python=["3.9"])
 def tests(session: Session) -> None:
-    args = session.posargs or ["--cov", "-m", "not e2e"]
+    args = session.posargs or ["--cov", "-m", "not e2e", "-vv"]
```

Comment:
Vi har ikke lyst på så mye logg når vi kjører deploy
```suggestion
    args = session.posargs or ["--cov", "-m", "not e2e"]
```